vnc.html

```diff
@@ -200,7 +200,7 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
                         <div class="noVNC_expander">Advanced</div>
                         <div><ul>
                             <li>
-                                <label><input id="noVNC_setting_true_color" type="checkbox" checked /> True Color</label>
+                                <label><input id="noVNC_setting_convert_color" type="checkbox" checked /> Convert Color</label>
```

Comment:
I'm not sure this makes sense. True Color is broken right now anyway, so I say keep this stuff as an internal thing which is only used for broken servers like the ATEN ones.